src/Enable.Extensions.Queuing.RabbitMQ/Internal/ConnectionManagement/RabbitMQManagedConnection.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using RabbitMQ.Client;
+
+namespace Enable.Extensions.Queuing.RabbitMQ.Internal
+{
+    internal class RabbitMQManagedConnection
+    {
+        public RabbitMQManagedConnection(IConnection connection)
+        {
+            Connection = connection;
+            Id = Guid.NewGuid();
+        }
+
+        public Guid Id { get; }
+
+        public IConnection Connection { get; private set; }
```

Comment:
Here we've explicitly used declared `private set`, but we haven't for the `Id` property. Can we make the properties in this class consistent and drop the `private set`s.

Reply:
The Id is meant to be readonly and therefore has no setter. The private sets for the Reference count and connection are there as the class sets these after the constructor.